src/test/groovy/com/homeaway/devtools/jenkins/testing/LocalProjectPipelineExtensionDetectorSpec.groovy

```diff
@@ -0,0 +1,18 @@
+package com.homeaway.devtools.jenkins.testing
+
+import com.homeaway.devtools.jenkins.testing.functions.ScriptToTest
+
+class LocalProjectPipelineExtensionDetectorSpec extends JenkinsPipelineSpecification {
+
+    def "LocalProjectPipelineExtensionDetectorSpec should detect classes that aren't direct subtypes of java-lang-Object" () {
```

Comment:
```suggestion
    def "LocalProjectPipelineExtensionDetector should detect classes that aren't direct subtypes of java-lang-Object" () {
```

It's really the LocalProjectPipelineExtensionDetector that you're testing, right?

If you were testing the `LoalProjectPipelineExtensionDetectorSpec`, you'd be checking the `DEFAULT_TEST_CLASSES` ( https://github.com/ExpediaGroup/jenkins-spock/blob/master/src/main/groovy/com/homeaway/devtools/jenkins/testing/JenkinsPipelineSpecification.groovy#L1137 ) variable's contents, right?

I think this test can actually just extend `Specification`, since it's testing a standalone, non-Jenkins-specific class (`LocalProjectPipelineExtensionDetector`), right?